docs/admin/install/serving/install-serving-with-yaml.md

```diff
@@ -293,24 +299,23 @@ Follow the procedure for the DNS of your choice:
             helloworld-go   http://helloworld-go.default.example.com   helloworld-go-vqjlf   helloworld-go-vqjlf   True
             ```
 
-    1. Instruct `curl` to connect to the External IP or CNAME defined by the
+    1. Instruct `curl` to connect to the External IP address or CNAME defined by the
       networking layer in section 3 above, and use the `-H "Host:"` command-line
-      option to specify the Knative application's host name. For example, if the
-      networking layer defines your External IP and port to be `http://192.168.39.228:32198` and you wish to access the above `helloworld-go` application, use:
+      option to specify the Knative application's hostname. For example, if the
```

Comment:
I'd say "hostname of the application"; I think it's better if we avoid attributing a possessive to inanimate things. Might be another good addition for the style guide if it's not already in the k8s or Google one.

Reply:
It's implicitly permitted per [Google Style](https://developers.google.com/style/possessives#examples-of-correct-possessives) -- note the "application" and "class" examples. According to that page, Google Style only forbids making product names into possessives, likely to avoid breaking trademark laws.

Granted, Google Style forbids [anthropomorphism](https://developers.google.com/style/anthropomorphism), but possession is not among the "human" examples. The concern seems to be more about avoiding full-blown metaphors.

I like assigning possession to inanimate objects because it makes sentences punchier. Also I've never heard of it confusing anyone. Which way do you lean, @snneji?

Reply:
sorry just passing through, this is the geekiest tech writer back-and-forth I've seen and I love it